Lesson Modules/Module 6- Data Binning.ipynb

```diff
@@ -98,8 +98,10 @@
     "            DataCount.append(0)\n",
     "            bin_num += 1\n",
     "    \n",
-    "    binnedCPM = np.array(sumCPM) / np.array(DataCount)              # np.array allows us to perform element-by-elemtent division\n",
-    "    binnedError = np.array(sumError) / np.array(DataCount)\n",
+    "    binnedCPM = np.array(sumCPM) / np.array(DataCount)          # np.array allows us to perform element-by-elemtent division\n",
+    "    \n",
+    "    avgError = np.array(sumError) / np.array(DataCount)\n",
+    "    stdError = avgError / np.sqrt(len(DataCount))               # standard error is average error divided by sqrt(# of elements)\n",
```

Comment:
Not `len(DataCount)`, but `DataCount` elementwise! That contains the number of measurements in each month. The division is elementwise because `avgError` is already an array and `np.sqrt` will return an array.